verl/utils/fsdp_utils.py

```diff
@@ -105,58 +107,61 @@ def lambda_policy_fn(module):
     return auto_wrap_policy
 
 
-def offload_fsdp_grad(module):
-    for _, param in module.named_parameters():
-        if param.grad is not None:
-            param.grad = param.grad.to("cpu", non_blocking=True)
-    torch.cuda.empty_cache()
-
-
-def load_fsdp_grad(module, device_id):
-    for _, param in module.named_parameters():
-        if param.grad is not None:
-            param.grad = param.grad.to(device_id, non_blocking=True)
-    torch.cuda.empty_cache()
-
-
-def offload_fsdp_param_and_grad(module, offload_grad=False):
-    for _, param in module.named_parameters():
-        if hasattr(param, "_local_shard"):
-            param._local_shard = param._local_shard.to("cpu", non_blocking=True)
-        param.data = param.data.to('cpu', non_blocking=True)
-        if offload_grad and param.grad is not None:
-            param.grad = param.grad.to("cpu", non_blocking=True)
-    torch.cuda.empty_cache()
+@torch.no_grad()
+def offload_fsdp_model_to_cpu(model: FSDP, empty_cache: bool = True):
```

Comment:
Shall we write a unit test for these two functions?

Reply:
Will add a CI in the future